cypress/commons/actions/brewery/BreweryParameters.js

```diff
@@ -128,6 +128,16 @@ const getCommentInput = () => ScenarioParameters.getParameterInput('text-input-c
 const getAdditionalDate = () => ScenarioParameters.getParameterValue('date-input-additional_date');
 const getAdditionalDateHelperText = () => cy.get('[id=date-text-field-additional_date-helper-text]');
 const getAdditionalDateInput = () => ScenarioParameters.getParameterInput('date-input-additional_date');
+const getScenarioToCompareInput = () => ScenarioParameters.getParameterInput('single-select-scenario_to_compare');
+const getScenarioToCompare = () => ScenarioParameters.getParameterValue('single-select-scenario_to_compare');
+
+const getScenarioToCompareSelect = () => {
+  return getScenarioToCompareInput();
+};
```

Comment:
Do we need a function that only calls another function?
Also, the 3 lines are equivalent to
`const getScenarioToCompareSelect = getScenarioToCompareInput;`